draft-ietf-sidrops-aspa-verification.xml

```diff
@@ -508,56 +508,27 @@ hop(AS(i), AS(j)) =  / Else, "Provider+" if the VAP-SPAS entry
-   <t>
-     While the ASPA-based AS_PATH verification method (<xref target="ingress"/>) detects and mitigates route leaks that were created by preceding ASes listed in the AS_PATH, 
-	 it lacks the ability to prevent route leaks from occuring at the local AS. 
-	 The use of the Only to Customer (OTC) Attribute <xref target="RFC9234"/> fills in that gap. 
-	 The procedures utilizing the OTC Attribute set out in <xref target="RFC9234"/> complement those described in this document. 
-	 Implementation of those procedures in addition to ASPA-based AS_PATH verification is encouraged.
-   </t>
-   </section>
+    <section anchor="ingress" title="Ingress eBGP Router vs. Internal BGP">
+      <t>ASPA can run on any BGP router. However, procedures for ASPA-based AS_PATH verification and mitigation described in this document are NOT RECOMMENDED for use on eBGP links internal to the Confederation.</t>
+    </section>
+
+    <section title="Address Families">
+      <t>The verification procedures described in this document MUST be applied to BGP routes with {AFI, SAFI} combinations {AFI 1 (IPv4), SAFI 1} and {AFI 2 (IPv6), SAFI 1} [IANA-AF]. The procedures MUST NOT be applied to other address families by default.</t>
+    </section>
+
+    <section title="Implementation of Verification Algorithm">
+      <t>Implementations of this specification are not required to implement the AS_PATH verification procedures exactly as described in Section XX but MUST provide functionality equivalent to the external behavior resulting from those procedures. Therefore, implementation may differ, for example, for computational efficiency purposes.</t>
+    </section>
+
+    <section anchor="mitigation" title="Configuration of a Route Leak Mitigation Policy">
+      <t>The specific configuration of mitigation policies based on AS_PATH verification using ASPA is at the discretion of the network operator. The following policies are highly recommended, though.</t>
+      <t><strong>Invalid</strong>: If the AS_PATH is determined to be Invalid, then the route SHOULD be considered ineligible for route selection (see Section 1.2). The route SHOULD not be considered further but MUST be kept in the Adj-RIB-In for potential future re-evaluation (see [RFC9324]).</t>
+      <t><strong>Valid and Unknown</strong>: Any Valid or Unknown AS_PATH SHOULD not be distinguished in the route selection process because of the ASPA validation outcome, but prefered over an Invalid AS_PATH.</t>
+      <t>For any route with an Invalid or Unknown AS_PATH, the cause of the Invalid or Unknown state SHOULD be logged for monitoring and diagnostic purposes. The cause of the Invalid state can be in the form of listing the AS hops which were evaluated by the hop-check function to be &quot;Not Provider+&quot;, or &quot;No Attestation&quot; or &quot;Not Provider+&quot; in case of Unkown. The logging router, however, cannot necessarily determine the origin of the route leak.</t>
```

Comment:
perhaps s/SHOULD be logged/MAY be logged/ - as logging the hop check errors is just noise

Reply:
s/Unkown/Unknown/

Reply:
MAY is not really helpful because logging is per se usually optional. ... the SHOULD is just a copy from the current version ("any route with an Unknown AS_PATH, the cause of the Unknown state SHOULD be logged for monitoring and diagnostic purposes.").

but i don't have strong feeling. we could also limit the SHOULD to Invalid.

Reply:
I think logging AS_PATH with unknown state is rather unhelpful since it will be the case for more than 99% of all updates for the foreseeable future.
From an Ops perspective I find such recommendations more then antiquated but I understand that this is just typical RFC boiler plate.